### PR TITLE
dap: remote attach must not use terminateDebuggee in single-client launch mode

### PR DESCRIPTION
**Pull request by polinasok**

### CODE REVIEW

File: service/dap/server.go [hunk 1]
```diff
@@ -1770,9 +1770,15 @@ func (s *Session) onAttachRequest(request *dap.AttachRequest) {
 		if s.config.Debugger.Backend == "rr" {
 			s.send(&dap.CapabilitiesEvent{Event: *newEvent("capabilities"), Body: dap.CapabilitiesEventBody{Capabilities: dap.Capabilities{SupportsStepBack: true}}})
 		}
-		// Give the user an option to terminate this server when client disconnects (default is to leave it)
-		s.send(&dap.CapabilitiesEvent{Event: *newEvent("capabilities"), Body: dap.CapabilitiesEventBody{Capabilities: dap.Capabilities{SupportTerminateDebuggee: true}}})
-		// TODO(polina); also use SupportSuspendDebuggee when available
+		// Customize termination options for debugger and debuggee
+		if s.config.AcceptMulti {
```
Comment (aarzilli) [inline service/dap/server.go:1774]: `if s.config.AcceptMulti || s.config.Debugger.AttachPid > 0`? Is there a reason to have it repeated?

Comment (polinasok) [reply service/dap/server.go:1774]: The differentiator is the TODO in that block. My next PR will implement it.

Comment (polinasok) [reply service/dap/server.go:1774]: Do you want me to change it in this PR?

**Review by aarzilli — APPROVED** [review]
LGTM

**Review by derekparker — APPROVED** [review]
LGTM

**Comment by derekparker** [comment]
Sorry haven't merged yet because of CI, re-running now.

File: service/dap/server_test.go [hunk 2]
```diff
@@ -6546,11 +6567,29 @@ func runTestWithDebugger(t *testing.T, dbg *debugger.Debugger, test func(c *dapt
 	client.ExpectTerminatedEvent(t)
 
 	<-serverStopped
+	if pid > 0 {
+		syscall.Kill(pid, syscall.SIGKILL)
```
Comment (aarzilli) [inline service/dap/server_test.go:6571]: syscall.Kill doesn't exist on windows, you have to use os.(*Process).Kill

Comment (polinasok) [reply service/dap/server_test.go:6571]: Ah, thanks for the pointer! Let me fix this.

**Comment by polinasok** [comment]
CI is very unhappy. I reran a bunch of times, but build failures are not going away. They don't seem related.

File: service/dap/server_test.go (hunk 2 above)
Comment (aarzilli) [reply service/dap/server_test.go]: It doesn't matter, it still needs to compile.

**Comment by aarzilli** [comment]
> CI is very unhappy. I reran a bunch of times, but build failures are not going away. They don't seem related.

It looks like things that should go away with a rebase.

**Comment by polinasok** [comment]
Rebase + extra rerun and everything is finally green!

**Pull request merged**